bin/utils/print-joblog-to-file.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project. 2020
+#######################################################################
+
+# Function: Print a JES job log to a USS file
+# Inputs  - jobname and jobid of job to be printed
+# Outputs - a USS file containing the job log, which will be placed in $jobname-$jobid.log
+#         - The job will be purged from the JES spool
+# Uses    - The TSO OUTPUT command
+
+# identify this script
+SCRIPT_DIR="$(dirname $0)"
+SCRIPT="$(basename $0)"
+echo script $SCRIPT started from $SCRIPT_DIR
+
+if [[ $# -ne 2 ]]
+then
+echo; echo $SCRIPT Usage:
+cat <<EndOfUsage
+
+    $SCRIPT jobname jobid
+
+for example
+
+    $SCRIPT myjob   job65262
+    $SCRIPT zwe1sv  stc00406
+
+The output will be placed in $jobname-$jobid.log and the job will be purged.
+EndOfUsage
+echo script $SCRIPT ended 
+exit 
+fi
+
+jobname=$1
+jobid=$2
+
+userid=${USER:-${USERNAME:-${LOGNAME}}}
+tsoJobname=RUNTSOCM
+tsoCommandOut=/tmp/print.job.log.$$.tso.out
```

Comment:
We should use the `TMPDIR` variable if set as we've had issues in the past we people who have changed from `/tmp`. The rest of our code base uses `${TMPDIR:-/tmp}` to get around this, so we should be consistent here.

Reply:
Happy to use TMPDIR, but as the support script is likely to be needed when things have gone wrong, and outside of the install process, so TMPDIR probably won't be set.

Reply:
TMPDIR is a system variable on unix, so I don't see what the install process has to do with it?

Reply:
OK, will use TMPDIR.